# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/RecordedEvent.h
# Commit: 908c5b3e65b4
# Full Hash: 908c5b3e65b491cfcd71acedc853f989191a3e22
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-12-09 21:50:19
# Regressor Bug: 1598582
# File Overlap Count: 1
# Description:
#   Bug 1598582 Part 2: Add event to record OptimizeSourceSurface. r=jrmuizel
#   
#   This also optimizes the surface returned by CreateSourceSurfaceFromData to
#   match what would happen in the non-recorded case. This means that all surfaces
#   of type RECORDING are optimized, i.e. they represent a surface type that
# ==============================================================================

diff -r 593bc922d369 -r 908c5b3e65b4 gfx/2d/RecordedEvent.h
--- a/gfx/2d/RecordedEvent.h	Thu Dec 05 15:07:06 2019 +0000
+++ b/gfx/2d/RecordedEvent.h	Fri Nov 29 21:59:11 2019 +0000
@@ -27,7 +27,7 @@
 const uint16_t kMajorRevision = 10;
 // A change in minor revision means additions of new events. New streams will
 // not play in older players.
-const uint16_t kMinorRevision = 0;
+const uint16_t kMinorRevision = 1;
 
 struct ReferencePtr {
   ReferencePtr() : mLongPtr(0) {}
@@ -319,6 +319,7 @@
     EXTERNALSURFACECREATION,
     FLUSH,
     DETACHALLSNAPSHOTS,
+    OPTIMIZESOURCESURFACE,
     LAST,
   };
 